core/src/saros/activities/TextSelectionActivity.java

```diff
@@ -46,15 +46,29 @@
   @XStreamAsAttribute
   private final int endInLineOffset;
 
+  @XStreamAlias("bs")
```

Comment:
Why is this not annotated with `@XStreamAlias`? Is there a reason for sending it as XML Tag instead of attribute ?

Reply:
I just forgot to add the `@XStreamAsAttribute`. Will do so.

Reply:
Done.